apps/right-to-rent-check/translations/src/en/pages.json

```diff
@@ -11,6 +11,13 @@
     "intro": "If the person has documents, you need to perform the check yourself.",
     "link": "<a href=\"https://www.gov.uk/check-tenant-right-to-rent-documents\">Back to guidance</a>"
   },
+  "check-not-needed": {
+    "link": "<a href=\"https://www.gov.uk/check-tenant-right-to-rent-documents\">Back to guidance</a>"
```

Comment:
this is and will be used in mutilple exit pages that's why I've called it check-not-needed as a generic name.  I've kept it also in an object for readability rather than
`check-not-needed-link: "<a href=\"https://www.gov.uk/check-tenant-right-to-rent-documents\">Back to guidance</a>"`

Reply:
I don't think html should be added to the translations file, only text.

Reply:
Yeah, you're right, that makes sense.